nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -442,20 +484,28 @@ def run_single_instance(config):
     config["reuse_actors"] = False
     config["dist_port"] = get_free_port()
 
+    # set trainable
+    ray_trainable = config.get("ray_trainable", SupervisedTrainable)
+    # assert issubclass(ray_trainable, BaseTrainable)
+
     # Build kwargs for `tune.run` function using merged config and command line dict
     kwargs_names = tune.run.__code__.co_varnames[:tune.run.__code__.co_argcount]
-    kwargs = dict(zip(kwargs_names, [SupervisedTrainable, *tune.run.__defaults__]))
+    kwargs = dict(zip(kwargs_names, [ray_trainable, *tune.run.__defaults__]))
     # Update`tune.run` kwargs with config
     kwargs.update(config)
     kwargs["config"] = config
     # Update tune stop criteria with config epochs
     stop = kwargs.get("stop", {}) or dict()
     epochs = config.get("epochs", 1)
-    stop.update(training_iteration=epochs)
+    num_tasks = config.get("num_tasks", None)
```

Comment:
This was also debugging code I believe

Reply:
Not really, this is required to run ContinualLearningExperiment in single instances. Arguably there are better ways of doing this, but it is fine as it is now.

Reply:
That's right, sorry. This works then